### PR TITLE
added package python3-Frozen-Flask

### PR DESCRIPTION
**Pull request by OHNONOTAMOTH**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by abenson** [comment]
Please fix the commit message to [follow our guidelines](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes) by [amending the commit](https://www.atlassian.com/git/tutorials/rewriting-history) and force pushing to this branch. (Please avoid closing this PR and opening a new PR.)

**Comment by OHNONOTAMOTH** [comment]
amended commit to satisfy xlint

File: srcpkgs/python3-Frozen-Flask/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgname=python3-Frozen-Flask
+version=1.0.2
+revision=1
+build_style=python3-pep517
+makedepends="hatch"
+depends="python3-Flask"
+short_desc="Library that can turn flask sites into static sites"
+maintainer="ohnonotamoth <dilanjohnson@outlook.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/Frozen-Flask/Frozen-Flask/"
```
Comment (Bnyro) [inline srcpkgs/python3-Frozen-Flask/template:11]:
```suggestion
homepage="https://github.com/Frozen-Flask/Frozen-Flask/"
changelog="https://github.com/Frozen-Flask/Frozen-Flask/releases"
```

File: srcpkgs/python3-Frozen-Flask/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-Frozen-Flask'
+pkgname=python3-Frozen-Flask
+version=1.0.2
+revision=1
+build_style=python3-pep517
+makedepends="hatch"
+depends="python3-Flask"
+short_desc="Library that can turn flask sites into static sites"
```
Comment (Bnyro) [inline srcpkgs/python3-Frozen-Flask/template:8]:
```suggestion
short_desc="Library that freezes Flask apps into static files"
```

**Pull request closed without merging**